src/lib.rs

```diff
@@ -343,21 +345,22 @@ trait Context: Sized {
     );
 }
 
-/// Instance of wgpu. First thing you create when using wgpu.
+/// Context for all other wgpu objects. Instance of wgpu.
 ///
-/// An instance sets up the context for all other wgpu objects.
+/// This is the first thing you create when using wgpu.
+/// Its primary use is to create [adapters](Adapter) and [surfaces](Surface).
 ///
-/// Instances and adapters do not have to be kept alive, only devices.
+/// Does not have to be kept alive.
 pub struct Instance {
     context: Arc<C>,
 }
 
 /// Handle to a physical graphics and/or compute device.
 ///
-/// An `Adapter` can be used to open a connection to the corresponding device on the host system,
-/// yielding a [`Device`] object.
+/// Adapters can be used to [open a connection][Adapter::request_device] to
+/// the corresponding [device][Device] on the host system.
```

Comment:
This link works as is (here and all other places). I would prefer to just use a raw link even if the case isn't right because it's in a inline code block, so it's clear it's a code reference.

Reply:
I've updated the PR to match the previous style